apps/artist/client/views/header.coffee

```diff
@@ -2,7 +2,7 @@ _ = require 'underscore'
 Backbone = require 'backbone'
 moment = require 'moment'
 { Following, FollowButton } = require '../../../../components/follow_button/index.coffee'
-{ CURRENT_ITEM } = require('sharify').data
+{ CURRENT_ITEM, SHOW_ARTIST_CTA } = require('sharify').data
```

Comment:
This is so we can intercept the follow button behavior.

Now, if the follow button is clicked when you're on an artist page with the CTA loaded, the behavior is modified (down below...)